linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_pam_timestamp_check/rule.yml

```
@@ -4,17 +4,25 @@ prodtype: rhcos4,ol7,ol8,rhel7,rhel8,rhv4,sle12,wrlinux1019
 
 title: 'Ensure auditd Collects Information on the Use of Privileged Commands - pam_timestamp_check'
 
+{{% if product in ["sle12", "sle15"] %}}
+{{% set pam_bin_path = "/sbin/pam_timestamp_check" %}}
+{{% else %}}
+{{% set pam_bin_path = "/usr/sbin//pam_timestamp_check" %}}
```

Double slash :scream: 
```suggestion
{{% set pam_bin_path = "/usr/sbin/pam_timestamp_check" %}}
```

---

Thank you and done.